app/_layouts/extension.html

```diff
@@ -178,19 +241,44 @@
   {% if config_required_fields_yaml == 'config: ' %}config:
     <optional_parameter>: <value>{% else %}{{ config_required_fields_yaml }}{% endif %}
 ```
+
+`<service>` is the `id` or `name` of the Service that this plugin
+  configuration will target.
+
+{% endnavtab %}
+{% endunless %}
+{% unless page.params.konnect_examples == false %}
+{% navtab Konnect %}
+Configure this plugin on a Service:
+
+1. In Konnect, select the service on the ServiceHub page.
+2. Scroll down to **Versions** and select the version.
+3. Scroll down to **Plugins** and click **New Plugin**.
+4. Find and select the **{{ page.name }}** plugin.
+
+    <div class="alert alert-ee blue">
+    <strong>Note:</strong> If the plugin is greyed out, then it is not available
+    for your product tier. See <a href="">Kong Gateway tiers</a>.
+    </div>
+
+  {% if config_required_fields_konnect == empty %}
+5. Click **Create** to save.
+  {% else %}  
+5. Enter the following required [parameters](#parameters), updating the default/sample values as needed:
+    {{ config_required_fields_konnect }}
+6. Click **Create** to save.
```

Comment:
"Click Create to save" sounds like there is an issue with GUI labels. Don't need the "to save" because it is actually creating the plugin, not saving an existing config.

```suggestion
6. Click **Create**.
```